src/js/stores/__tests__/MesosStateStore-test.js

```diff
@@ -142,12 +142,22 @@ describe('MesosStateStore', function () {
 
     it('should find a currently running task', function () {
```

Comment:
What do you think about adding a test to check that we get an `instanceof` `Task` struct?